admin_manual/configuration/server/background_jobs_configuration.rst

```diff
@@ -122,13 +119,10 @@ While not exhaustive, these include:
 CleanupChunks
 ^^^^^^^^^^^^^
 
-The CleanupChunks job, contained in ``OCA\\DAV\\Command\\CleanupChunks``, will clean up outdated chunks (uploaded files) more than a certain number of days old.
-By default, the command cleans up chunks more than two days old. However, the command also accepts the number of days as an argument.
-It can be run, as follows, using the OCC command
+The CleanupChunks job command, contained in ``OCA\\DAV\\Command\\CleanupChunks``, will clean up outdated chunks (uploaded files) more than a certain number of days old and needs to be added to your crontab.
```

Comment:
I think an admin doesn't need to know about `OCA\\DAV\Commend\\CleanupChunks` but rather bout `occ dav:cleanup-chunks`

Replace this with "The CleanupChunks command ``occ dav:cleanup-chunks`` will clean up outdated ..."

Reply:
Shall do. Then it's good to go?